### DISCUSSION TITLE
Ideas for Prototype V1.0

### DISCUSSION THREAD

**Discussion by Mister-Mario** [Arquisoft/wiq_en1b#53] [Ideas]
Here I will post the ideas that I think are needed for the next prototype in order of importance:

1. Improving the distractors
2. Implementing the login and registering logic
3. Creating the database for saving users information
4. Migrating the backend to express
5. Creating the database to load the questions and recover them in the backend
6. Creating the historical data of the users
7.Checking the correctness of answers
8. Making the question timer work
9. Making the answers show in random places
10. Showing the historical data of the user
11. Making an open API?

**Comment by ErdemYabaci** [comment]
I can keep working on distractors, though I think we need to find a way to get pages with wanted properties instead of hard-coding pages that we know have them for true randomness. Jorge and I started talking about it last lesson, I'll add a comment if we reach a conclusion on that part.
